tasks/1-duplicates/duplicates.py

```diff
@@ -5,18 +5,21 @@
 
 def remove_duplicates(input: List[str]) -> List[str]:
     """This method should return a new list with duplicates removed."""
-
-    output = input  # Replace with your logic
-
-    return output
+    deduped_list = []
+    for name in example_list:
+        if name not in deduped_list:
+            deduped_list.append(name)
+    return deduped_list
+   
 
 
 def get_duplicates(input: List[str]) -> List[str]:
     """This method should return a list of the items which were duplicated."""
-
-    output = input  # Replace with your logic
-
-    return output
+    list_seen = set()
+    list_added = list_seen.add
+    list_of_duplicates = set(name for name in example_list if name in list_seen or list_added(name))
```

Comment:
Interesting... does `set.add()` return true if it is already present?

This was a tricky one. I can't even remember how I solved it.